garage/tf/algos/ddpg.py

```diff
@@ -20,50 +11,78 @@
 
 
 class DDPG(OffPolicyRLAlgorithm):
-    """
-    A DDPG model based on https://arxiv.org/pdf/1509.02971.pdf.
+    """A DDPG model based on https://arxiv.org/pdf/1509.02971.pdf.
```

Comment:
the original position of this documentation (module) was fine. but this is fine too. just please be consistent.